pom.xml

```diff
@@ -416,7 +416,7 @@
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
         <carbon.kernel.version>4.6.0</carbon.kernel.version>
-        <identity.governance.version>1.3.39</identity.governance.version>
+        <identity.governance.version>1.4.24-SNAPSHOT</identity.governance.version>
```

Comment:
The proper released version needs to be added once https://github.com/wso2-extensions/identity-governance/pull/405 is merged and a new version is released.

Reply:
let's not add the snapshot version, as there can be a mistakenly merge causing problems. Let's keep the PR in draft mode without the version update since the dependencies are met

Reply:
fixed: a8a1fba1a6e787f41f6566d0765bb47f6ce4c73e